lib/module.js

```diff
@@ -26,10 +26,14 @@ const generateHeaderFile = (context, resources) => {
   const headersFile = `${generateDir}/_headers`;
   let content = '/*\n';
   resources.forEach(resource => {
-    const files = glob.sync(`${generateDir}/${resource.path}`);
+    const files = glob.sync(`${resource.path}`);
     files.forEach(file => {
       const url = file.replace(generateDir, '');
```

Comment:
@derevandal You use WIndows, right?

It seems that this line doesn't work as expected on Windows. Could you check the values of `generateDir`, `files` and `url`?

Reply:
I'll test it and post the log here.
Nevertheless, Netlify doesn't push the assets even with `_headers` automatically generated during `yarn generate`. :(

Reply:
@jmblog these are the values that you asked for:
```sh
generateDir: C:\Users\andre.vandal\dev\virada-tecnospeed\dist
resource: {
  path: '**/*.js',
  as: 'script'
}
file: C:/Users/derevandal/dev/test/dist/assets/0878cfcff900ea74e8df.js
url: C:/Users/derevandal/dev/test/dist/assets/0878cfcff900ea74e8df.js 
file: C:/Users/derevandal/dev/test/dist/assets/224cf76102929e9a955a.js
url: C:/Users/derevandal/dev/test/dist/assets/224cf76102929e9a955a.js 
file: C:/Users/derevandal/dev/test/dist/assets/26ac87831e199474822f.js
url: C:/Users/derevandal/dev/test/dist/assets/26ac87831e199474822f.js 
file: C:/Users/derevandal/dev/test/dist/assets/4de967c7be1170f297bc.js
url: C:/Users/derevandal/dev/test/dist/assets/4de967c7be1170f297bc.js 
file: C:/Users/derevandal/dev/test/dist/assets/5bb713b0e5d125922aa4.js
url: C:/Users/derevandal/dev/test/dist/assets/5bb713b0e5d125922aa4.js 
file: C:/Users/derevandal/dev/test/dist/assets/a3dcd05bbe2da6f3be6f.js
url: C:/Users/derevandal/dev/test/dist/assets/a3dcd05bbe2da6f3be6f.js 
file: C:/Users/derevandal/dev/test/dist/assets/a85485e574526399ce46.js
url: C:/Users/derevandal/dev/test/dist/assets/a85485e574526399ce46.js 
file: C:/Users/derevandal/dev/test/dist/assets/b7d7abb2f34e621a0cb9.js
url: C:/Users/derevandal/dev/test/dist/assets/b7d7abb2f34e621a0cb9.js 
file: C:/Users/derevandal/dev/test/dist/assets/c49ca96955f1359e0275.js
url: C:/Users/derevandal/dev/test/dist/assets/c49ca96955f1359e0275.js 
file: C:/Users/derevandal/dev/test/dist/assets/e2c742de3d7699b88134.js
url: C:/Users/derevandal/dev/test/dist/assets/e2c742de3d7699b88134.js 
file: C:/Users/derevandal/dev/test/dist/assets/workbox.4c4f5ca6.js    
url: C:/Users/derevandal/dev/test/dist/assets/workbox.4c4f5ca6.js     
file: C:/Users/derevandal/dev/test/dist/sw.js 
url: C:/Users/derevandal/dev/test/dist/sw.js  
resource: {                                   
  path: '/fonts/hinted-GothamRounded-Book.woff2',
  as: 'fonts',
  crossorigin: true
}
file: C:/Users/derevandal/dev/test/dist/fonts/hinted-GothamRounded-Book.woff2
url: C:/Users/derevandal/dev/test/dist/fonts/hinted-GothamRounded-Book.woff2 
resource: {                                   
  path: '/fonts/hinted-GothamRounded-Bold.woff2',
  as: 'fonts',
  crossorigin: true
}
file: C:/Users/derevandal/dev/test/dist/fonts/hinted-GothamRounded-Bold.woff2
url: C:/Users/derevandal/dev/test/dist/fonts/hinted-GothamRounded-Bold.woff2 
resource: {                                   
  path: '/images/*.svg',
  as: 'image'
}
file: C:/Users/derevandal/dev/test/dist/images/boleto.svg             
url: C:/Users/derevandal/dev/test/dist/images/boleto.svg              
file: C:/Users/derevandal/dev/test/dist/images/cte.svg                
url: C:/Users/derevandal/dev/test/dist/images/cte.svg                 
file: C:/Users/derevandal/dev/test/dist/images/cteos.svg              
url: C:/Users/derevandal/dev/test/dist/images/cteos.svg               
file: C:/Users/derevandal/dev/test/dist/images/dots.svg               
url: C:/Users/derevandal/dev/test/dist/images/dots.svg                
file: C:/Users/derevandal/dev/test/dist/images/esocial.svg            
url: C:/Users/derevandal/dev/test/dist/images/esocial.svg             
file: C:/Users/derevandal/dev/test/dist/images/gnre.svg               
url: C:/Users/derevandal/dev/test/dist/images/gnre.svg                
file: C:/Users/derevandal/dev/test/dist/images/mdfe.svg               
url: C:/Users/derevandal/dev/test/dist/images/mdfe.svg                
file: C:/Users/derevandal/dev/test/dist/images/nfce.svg               
url: C:/Users/derevandal/dev/test/dist/images/nfce.svg                
file: C:/Users/derevandal/dev/test/dist/images/nfe.svg                
url: C:/Users/derevandal/dev/test/dist/images/nfe.svg                 
file: C:/Users/derevandal/dev/test/dist/images/nfse.svg               
url: C:/Users/derevandal/dev/test/dist/images/nfse.svg                
file: C:/Users/derevandal/dev/test/dist/images/notasegura.svg         
url: C:/Users/derevandal/dev/test/dist/images/notasegura.svg          
file: C:/Users/derevandal/dev/test/dist/images/reinf.svg              
url: C:/Users/derevandal/dev/test/dist/images/reinf.svg               
file: C:/Users/derevandal/dev/test/dist/images/sat.svg                
url: C:/Users/derevandal/dev/test/dist/images/sat.svg                 
file: C:/Users/derevandal/dev/test/dist/images/sped.svg               
url: C:/Users/derevandal/dev/test/dist/images/sped.svg                
file: C:/Users/derevandal/dev/test/dist/images/tecnospeed.svg         
url: C:/Users/derevandal/dev/test/dist/images/tecnospeed.svg          
file: C:/Users/derevandal/dev/test/dist/images/virada.svg             
url: C:/Users/derevandal/dev/test/dist/images/virada.svg
```